lld/test/ELF/aarch64-ptrauth.s

```diff
@@ -0,0 +1,156 @@
+// REQUIRES: aarch64
```

Comment:
Why isn't this named as `aarch64-feature-*`?

Reply:
We can probably rename it to aarch64-reloc-pauth.s or smth like that (there are already tests for other archs with such naming convention).

aarch64-feature-pauth.s checks the `.note.AARCH64-PAUTH-ABI-tag` section stuff.

They can be combined into one file, but IMHO testing different things in different relatively small files makes things easier to understand - even when combined, the files would be effectively just concatenated.

Reply:
I also find it easier when test files are small to understand things when a test fails.
OTOH, many small test files (and therefore, more RUN lines, more process invocations) results in slower overall run time of the regression tests, which does slow down all LLVM developers many times per day...

Not sure if anyone has tried to come up with a solution to both (a) have few process invocations due to RUN lines across many test files; and (b) easily be able to run small test files when a test breaks.

Reply:
Renamed aarch64-ptrauth.s to aarch64-reloc-pauth.s to make naming consistent with existing tests, aarch64-feature-pauth.s checks the `.note.AARCH64-PAUTH-ABI-tag` stuff. See b791da9dd02f8b3bf59d2d235181f945844d3039.